test/unit/modules/accounts.js

```diff
@@ -0,0 +1,725 @@
+				'-5d28e992b80172f38d3a2f9592cad740fd18d3c2e187745cd5f7badf285ed819'
+			];
+
+			it('should return error if invalid passpharse', function (done) {
+				account.shared.addDelegates({
+					body: {
+						publicKey: node.gAccount.publicKey,
+						secret: node.eAccount.password,
+						delegates: votes,
+					}
+				}, function (err, res){
+					expect(err).to.equal('Invalid passphrase');
+					done();
+				});
+			});
+
+			it('should throw if parameter doesnt have correct schema', function (done) {
+				account.shared.addDelegates({
+					body: {
+						secret: 5
+					}
+				}, function (err, res){
+					expect(err).to.equal('Expected type string but found type integer');
+					done();
+				});
+			});
+
+			it('should add delegate votes for an account', function (done) {
+				account.shared.addDelegates({
+					body: {
+						publicKey: node.gAccount.publicKey,
+						secret: node.gAccount.password,
+						delegates: votes
+					}
+				}, function (err, res){
+					expect(err).to.not.exist;
+					expect(res).to.be.an('object');
+					expect(res.transaction.asset.votes).to.eql(votes);
+					done();
+				});
+			});
+		});
+
+		describe('addDelegates (multisignature account)', function () {
+
+			function postSignature (transaction, signature, done) {
+				node.post('/peer/signatures', {
+					signature: {
+						transaction: transaction.id,
+						signature: signature
+					}
+				}, done);
+			}
+
+			function addTransaction (transaction, done) {
+				node.post('/peer/transactions', {
+					transaction: transaction
+				}, function (err, res) {
+					done(err, res.body);
+				});
+			}
+
+			var multiAccount = node.randomAccount();
+			var multiAccount2 = node.randomAccount();
+			var multiAccount3 = node.randomAccount();
+
+			before(function (done) {
+				var transferTrs = node.lisk.transaction.createTransaction(multiAccount.address, 100000000000, node.gAccount.password);
+				addTransaction(transferTrs, node.onNewBlock.bind(node, function (err) {
+					expect(err).to.not.exist;
+					var lifetime = 1;
+					var min = 2;
+					var keysgroup = [
+						'+' + multiAccount2.publicKey,
+						'+' + multiAccount3.publicKey
+					];
+
+					var multiTrs = node.lisk.multisignature.createMultisignature(multiAccount.password, null, keysgroup, lifetime, min);
+					addTransaction(multiTrs, function (err, res) {
+						expect(err).to.not.exist;
+						var signature1 = node.lisk.multisignature.signTransaction(multiTrs, multiAccount2.password);
+						var signature2 = node.lisk.multisignature.signTransaction(multiTrs, multiAccount3.password);
+						async.each([signature1, signature2], function (signature, eachCb) {
+							postSignature(multiTrs, signature, eachCb);
+						}, function (err) {
+							expect(err).to.not.exist;
+							node.onNewBlock(done);
+						});
+					});
+				}));
+			});
+
+			var votes = [
+				'+141b16ac8d5bd150f16b1caa08f689057ca4c4434445e56661831f4e671b7c0a',
```

Comment:
Minor issue, can we provide a comment on which accounts (username) these votes are going to?